deploy/alibaba-cloud/README.md

```diff
@@ -0,0 +1,151 @@
+### Install triton's python client
+```
+pip3 install nvidia-pyindex
+pip install tritonclient[all]
+```
+### A demo to query inception model
+```
+import numpy as np
+import time
+from PIL import Image
+
+import tritonclient.http as httpclient
+from tritonclient.utils import InferenceServerException
+
+URL = "<servcice url>"
+HEADERS = {"Authorization": "<service token>"}
+input_img = httpclient.InferInput("input", [1, 299, 299, 3], "FP32")
+img = Image.open('./cat.png').resize((299, 299))
+img = np.asarray(img).astype('float32') / 255.0
+input_img.set_data_from_numpy(img.reshape([1, 299, 299, 3]), binary_data=True)
+
+output = httpclient.InferRequestedOutput(
+    "InceptionV3/Predictions/Softmax", binary_data=True
+)
+triton_client = httpclient.InferenceServerClient(url=URL, verbose=False)
+
+start = time.time()
+for i in range(10):
+    results = triton_client.infer(
+        "inception_graphdef", inputs=[input_img], outputs=[output], headers=HEADERS
+    )
+    res_body = results.get_response()
+    elapsed_ms = (time.time() - start) * 1000
+    if i == 0:
+        print("model name: ", res_body["model_name"])
+        print("model version: ", res_body["model_version"])
+        print("output name: ", res_body["outputs"][0]["name"])
+        print("output shape: ", res_body["outputs"][0]["shape"])
+    print("[{}] Avg rt(ms): {:.2f}".format(i, elapsed_ms))
+    start = time.time()
+```
+You will get the following result by running the python script:
+```
+[0] Avg rt(ms): 86.05
+[1] Avg rt(ms): 52.35
+[2] Avg rt(ms): 50.56
+[3] Avg rt(ms): 43.45
+[4] Avg rt(ms): 41.19
+[5] Avg rt(ms): 40.55
+[6] Avg rt(ms): 37.24
+[7] Avg rt(ms): 37.16
+[8] Avg rt(ms): 36.68
+[9] Avg rt(ms): 34.24
+[10] Avg rt(ms): 34.27
+```
+# Additional Resources
+See the following resources to learn more about how to use Alibaba Cloud's OSS orEAS.
+- [Alibaba Cloud OSS's Document](https://help.aliyun.com/product/31815.html?spm=a2c4g.11186623.6.540.3c0f62e7q3jw8b)
+
+
+# Known Issues
+- [Binary Tensor Data Extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md) is not fully supported yet, users want to use service with binary extension supported, it is only available in cn-shanghai region of PAI-EAS.
+- Currently only HTTP/1 is supported, hence gRPC cannot be used when query Triton servers on EAS. HTP/2 will be officially supported in a short time.
+- Users should not mount a whole OSS bucket when launching Triton processor, but an arbitrarily deep sub-directory in bucket. Otherwise the mounted path will no be as expected.
+- Not all of Triton Server parameters are be supported on EAS, the following params are the ones supported now.
+```
+log-verbose
+log-info
+log-warning
+log-error
+exit-on-error
+strict-model-config
+strict-readiness
+allow-http
+http-thread-count
+pinned-memory-pool-byte-size
+cuda-memory-pool-byte-size
+min-supported-compute-capability
+buffer-manager-thread-count
+backend-config
+```
```

Comment:
Was `model-repository` omitted intentionally?

Reply:
It's not intentionally, I add it now.

Reply:
I do not see model-repository yet

Reply:
It is in line 137 of current file.